cmd/osbuild-composer/main.go

```diff
@@ -57,6 +59,19 @@ func createTLSConfig(c *connectionConfig) (*tls.Config, error) {
 	}, nil
 }
 
+func firstListener(listeners map[string][]net.Listener, name string) (net.Listener, error) {
```

Comment:
Why is this named firstListener? Seems like getListener would be clearer.

Reply:
It returns the first listener. The `activation` API is a bit cumbersome in that regard. systemd socket units can have multiple listeners, and this API returns all of them by name. This function is meant to only return the first of them,